app/controllers/components/course/forums_component.rb

```diff
@@ -36,7 +36,7 @@ def settings_sidebar_items
   end
 
   def unread_count
-    # :TODO
-    0
+    return 0 if current_course.nil? || current_user.nil?
```

Comment:
will current_course or current_user be nil? I thought cancancan would have 403'ed the page.